epuck_ros2_cpp/src/driver.cpp

```diff
@@ -263,6 +317,41 @@ class EPuckPublisher : public rclcpp::Node {
     mImuPublisher->publish(msg);
   }
 
+  void publishGroundSensorData() {
+    mI2cMain->setAddress(GROUND_SENSOR_ADDRESS);
+
+    for (int i = 0; i < 3; i++) {
+      int16_t raw = (mI2cMain->readInt8Register(2 * i) << 8) | mI2cMain->readInt8Register(2 * i + 1);
+
+      float k = -0.016 / (1000 - 300);
```

Comment:
```suggestion
      const float k = -0.016 / (1000 - 300);
```

Reply:
If you can find more self-explanatory name for this variable it would be nice (it is not easy to understand quickly what it does represent).